sycl/include/CL/sycl/handler.hpp

```diff
@@ -1231,31 +1248,26 @@ class __SYCL_EXPORT handler {
   }
 
   template <typename FuncT>
-  detail::enable_if_t<detail::check_fn_signature<
-      detail::remove_reference_t<FuncT>, void()>::value>
-  codeplay_host_task(FuncT Func) {
-    throwIfActionIsCreated();
-
-    MNDRDesc.set(range<1>(1));
-    MArgs = std::move(MAssociatedAccesors);
-
-    MHostTask.reset(new detail::HostTask(std::move(Func)));
-
-    setType(detail::CG::CODEPLAY_HOST_TASK);
-  }
-
+  __SYCL2020_DEPRECATED(
+      "codeplay_host_task() is deprecated, use host_task() instead")
+  detail::enable_if_t<
+      detail::check_fn_signature<detail::remove_reference_t<FuncT>,
+                                 void()>::value ||
+      detail::check_fn_signature<
+          detail::remove_reference_t<FuncT>,
+          void(interop_handle)>::value> codeplay_host_task(FuncT Func) {
+    host_task_impl(Func);
+  }
+
+  /// Enqueues a command to the SYCL runtime to invoke \p Func once.
```

Comment:
Is this comment describing enough?

Reply:
Thank you for the code-review!
I copied the comment from SYCL-2020 SPEC, which does not give any additional info other than this.